compiler/z/codegen/SystemLinkagezOS.cpp

```diff
@@ -186,7 +186,7 @@ TR::S390zOSSystemLinkage::S390zOSSystemLinkage(TR::CodeGenerator* cg)
    setOffsetToRegSaveArea(2048);
    setOffsetToLongDispSlot(0);
    setOffsetToFirstLocal(0);
-   setOutgoingParmAreaBeginOffset(getOffsetToFirstParm() - XPLINK_STACK_FRAME_BIAS);
+   setOutgoingParmAreaBeginOffset(self()->getOffsetToFirstParm() - XPLINK_STACK_FRAME_BIAS);
```

Comment:
This `self()` call is inside a constructor I think.

Reply:
Fixed in [force-push](https://github.com/eclipse/omr/compare/7960c05c62582d1a243ac0b279522f13225a4a35..4d776fa9292408995bfc4b357cf091e0f9b56389).